framework_tvb/tvb/adapters/exporters/tvb_export.py

```diff
@@ -34,18 +34,21 @@
 
 import os
 from tvb.adapters.exporters.abcexporter import ABCExporter
-from tvb.core.entities.file.files_helper import FilesHelper
 from tvb.adapters.exporters.exceptions import ExportException
 from tvb.core.entities.model.model_datatype import DataType
 from tvb.core.neocom import h5
 from tvb.core.neotraits.h5 import H5File
+from tvb.storage.storage_interface import StorageInterface
 
 
 class TVBExporter(ABCExporter):
     """ 
     This exporter simply provides for download data in TVB format
     """
     OPERATION_FOLDER_PREFIX = "Operation_"
```

Comment:
This could also be moved inside tvb_storage.

Reply:
What? The TVBExporter class or the OPERATION_FOLDER_PREFIX constant?

Reply:
OPERATION_FOLDER_PREFIX